test/srt/openai/test_serving_completions.py

```diff
@@ -52,141 +47,25 @@ def mock_tokenizer_manager():
 @pytest.fixture
 def completion_handler(mock_tokenizer_manager):
     """Create a completion handler instance"""
-    return CompletionHandler(mock_tokenizer_manager)
+    return OpenAIServingCompletion(mock_tokenizer_manager)
 
 
 class TestUtilityFunctions:
```

Comment:
Same. We can move the test for create error response for now. Move it to test_serving_base.py later